src/Web/Tailwind.hs

```diff
@@ -101,26 +113,21 @@ instance Default Css where
       |]
 
 instance Text.Show.Show TailwindConfig where
-  show (decodeUtf8 . encode -> config) =
-    -- Use `Object.assign` to merge JSON (produced in Haskell) with the rest of
-    -- config (defined by raw JS; that cannot be JSON encoded)
-    toString
-      [text|
-      module.exports =
-        Object.assign(
-          ${config},
-          {
-            theme: {
-              extend: {},
-            },
-            plugins: [
-              require('@tailwindcss/typography'),
-              require('@tailwindcss/forms'),
-              require('@tailwindcss/line-clamp'),
-              require('@tailwindcss/aspect-ratio')
-            ],
-          })
-      |]
```

Comment:
This PR looks good to me overall, except - why are we throwing away this template and writing it from scratch? You can just splice in the plugin list as we do with `${config}`.

Reply:
Oh, wait, now I see what is happening. Earlier I thought ToJSON is not used, but you see it *is* being used here. See line 104, `decodeUtf8 . encode -> config`. So the right approach here is to add back that `CustomJSON` instance and remove the `plugins` thing here (because it will now come from `${config}` automatically).

Reply:
The new code will basically look like this:

```haskell
    toString
      [text|
      module.exports =
        Object.assign(
          ${config},
          {
            theme: {
              extend: {},
            }
          })
      |]
```